### PR TITLE
Added status messages to console output for dedicated servers

### PR DESCRIPTION
**Pull request by Mailaender**
as wished in https://github.com/OpenRA/OpenRA/issues/6913.

### CODE REVIEW

**Comment by Mailaender** [comment]
This may be easier to review with https://github.com/OpenRA/OpenRA/pull/7851/files?w=1

**Comment by pchote** [comment]
This logs the chat responses to lobby commands, which isn't really what #6913 was asking for.

**Comment by Mailaender** [comment]
This should be still useful in itself even if it does not close #6913.

**Comment by penev92** [comment]
I think this can use @baxxxster's opinion.

**Comment by baxxxster** [comment]
Initially what was wanted, was some more details regarding the launch of a server, First time I launched a server I didn't understand that "Launching mod: ra" ment it loaded, I'd want more like "Mod loaded successfully". But imo, all servers should have info about players joining, leaving, chatlog preferably (IP should be displayed on join only to the server window, this way a server owner can ban users that misbehave)

File: OpenRA.Game/Server/Server.cs [hunk 1]
```diff
@@ -444,6 +444,9 @@ public void SendOrderTo(Connection conn, string order, string data)
 		public void SendMessage(string text)
 		{
 			DispatchOrdersToClients(null, 0, new ServerOrder("Message", text).Serialize());
+
+			if (Settings.Dedicated)
+				Console.WriteLine(text);
```
Comment (obrakmann) [inline OpenRA.Game/Server/Server.cs]: Maybe add a timestamp in front?

**Comment by obrakmann** [comment]
While this doesn't close #6913, I think it at least helps with getting an idea of what happens on a server while it runs, so :+1:, even without the timestamp.

**Comment by pchote** [comment]
I'll be okay with this if you can remove the "closes #6913" from the description of 14079eb and add the timestamps that @obrakmann requested.

**Comment by baxxxster** [comment]
Would be nice if it was possible for the servers to log to a file as well, I for instance like to create websites where I open logfiles and parse them on my own :)

**Comment by baxxxster** [comment]
Another handy feature with logfile for the server would be to be able to get the state of the server, if you want to do a restart of the server every X hours, it could be nice to figure out from the logfile.

Logging to a file would preferably be something like "logs/$hostname-timestamp.log" or something, as you can run multiple instances from same install

**Comment by ihptru** [comment]
one of the best identifiers for server can be PORT and can be handled in 1 log file

**Comment by abcdefg30** [comment]
Needs a rebase.

**Comment by penev92** [comment]
This has been put on the backburner in favor of the Coverity fixes, I suppose.
Closing until you can get back to it.

File: OpenRA.Game/Server/Server.cs (hunk 1 above)
Comment (Mailaender) [inline OpenRA.Game/Server/Server.cs]: Good idea. Done.

**Comment by Mailaender** [comment]
Rebased.

**Comment by penev92** [comment]
@obrakmann's comment sums it up for me:
>While this doesn't close #6913, I think it at least helps with getting an idea of what happens on a server while it runs, so :+1:

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/0ca45bf850135d12bcc61aedd44ba928ae4443d1...b232a7de8e2b4c3c694d5e3daba3c0242961ac4a)

**Pull request merged**